CHANGELOG.md

```diff
@@ -8,9 +8,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Example use case of spec-generated Ruby client. 
   [cyberark/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
-- The /whoami endpoint is now included in the spec. Allows for requesting info about the current client.
+- The `/whoami` endpoint is now included in the spec. Allows for requesting info about the current client.
   [cyberark/conjur-openapi-spec#56](https://github.com/cyberark/conjur-openapi-spec/issues/56)
 - Authenticator status endpoint is now included in the spec file. Allows checking if certain authenticators are working.
   [cyberark/conjur-openapi-spec#59](https://github.com/cyberark/conjur-openapi-spec/issues/59)
 - `/resources` endpoint now included in the OpenAPI specification.
   [cybeark/conjur-openapi-spec#62](https://github.com/cyberark/conjur-openapi-spec/issues/62)
+- The `/accounts` endpoint is now included in the spec file. Allows users to check which authenticators are installed and enabled.
```

Comment:
is `accounts` the right name here?

Reply:
It definitely is not. Don't know what I was thinking there